pkg/alertmanager/alertmanager_test.go

```diff
@@ -345,13 +343,16 @@ func TestSilenceLimits(t *testing.T) {
 		Store:             prepareInMemoryAlertStore(),
 		Replicator:        &stubReplicator{},
 		ReplicationFactor: 1,
-		// Set the interval to 1s as this test can trigger multiple broadcasts
-		// creating and expiring silences.
-		PersisterConfig: PersisterConfig{Interval: time.Second},
+		// We have set this 1 minute, but we don't use it in this
```

Comment:
If we don't use the interval, why do we have to change the current value? Or even better, why do we have to set it?

Reply:
Good question. Most tests use 1 hour for the interval, and this test used to use 1 hour before I changed it to 1 second, thinking the interval was the issue. However, I found that the interval is not the issue, so I increased it from 1 second to 1 minute. I can also change it back to 1 hour which will be a no-op – but at least consistent with the other tests.

Reply:
Is it possible not to specify the interval length at all, since it is not used? If it is not possible, then I would set it to 1h in order to be consistent with other tests, and I would write a comment stating that we set it to 1h for consistency, but that actually it will be overridden by a call to Setxxx. WDYT?

Reply:
It's not possible I'm afraid, otherwise it panics with `panic: non-positive interval for NewTicker`.  I've set it to 1h 👍

There is an existing comment:

> // We have set this to 1 hour, but we don't use it in this test as we override the broadcast function with SetBroadcast.

Is that sufficient?

Reply:
Thank you